### ISSUE TITLE
The ball changing position on replay with ' fake kick '

### ISSUE THREAD

**Issue by imsinanulgen** [haxball/haxball-issues#1249]
Hi,

I always thought we cant see fake kicks on replay because the replay system works with 0 extrapolation but yesterday i watched some replays and when i tried to watch with 0.1x i found that.

I just wondered is it normal for Haxball Replays? I discovered newly and i wanted to share with u guys.

Please watch with 0.1x speed 20:41 : https://thehax.pl/forum/powtorki.php?nagranie=eb5240d6e7df4469dd2fc59f048de590
MP4 version : https://streamable.com/j2tz8j

**Comment by kingjasko** [comment]
yeah, it's "normal" as in it does show on replayer, i noticed it on recs probably more than a year ago, similarly to how you can fake x on 0 extra, you can also see it on rec if it's a perfect fake x
